comictaggerlib/autotagmatchwindow.py

```diff
@@ -247,10 +247,10 @@ def save_match(self) -> None:
 
         QtWidgets.QApplication.setOverrideCursor(QtGui.QCursor(QtCore.Qt.CursorShape.WaitCursor))
         md.overlay(ct_md)
-        success = ca.write_metadata(md, self._style)
-        ca.load_cache(list(metadata_styles))
-
-        QtWidgets.QApplication.restoreOverrideCursor()
+        for style in self._styles:
+            success = ca.write_metadata(md, style)
+            QtWidgets.QApplication.restoreOverrideCursor()
+            if not success:
+                QtWidgets.QMessageBox.warning(self, "Write Error", "Saving the tags to the archive seemed to fail!")
```

Comment:
This needs to break on failure

Reply:
I'll add the style name too otherwise it's going to be too vague.